ale_linters/tex/textidote.vim

```diff
@@ -0,0 +1,47 @@
+" Author: Jordi Altayo <jordiag@kth.se>, Juan Pablo Stumpf
+" <juanolon@gmail.com>
+" Description: support for textidote grammar and syntax checker
+
+call ale#Set('tex_textidote_executable', 'textidote')
+call ale#Set('tex_textidote_options', '--no-color --output singleline')
+" TODO get language from spell spelllang
+call ale#Set('tex_textidote_check_lang', '')
+
+
+function! ale_linters#tex#textidote#GetExecutable(buffer) abort
+    let l:exe = ale#Var(a:buffer, 'tex_textidote_executable')
+    let l:exe .= ' ' . ale#Var(a:buffer, 'tex_textidote_options')
+
+    let l:check_lang = ale#Var(a:buffer, 'tex_textidote_check_lang')
+
+    if !empty(l:check_lang)
+        let l:exe .= ' --check ' . l:check_lang
+    endif
+
+    return l:exe . ' ' . expand('#' . a:buffer . ':t')
```

Comment:
This does not work when editing files in sub-foders. For example:

```
main.tex
chapter1/section01.tex
```

Removing the ':t' would properly show the textidote reported errors on both the main.tex and the chapter1/section01.tex files.